src/reports/report-html-generator.tsx

```diff
@@ -50,6 +50,10 @@ export class ReportHtmlGenerator {
                 getGuidanceTagsFromGuidanceLinks: this.getGuidanceTagsFromGuidanceLinks,
                 getPropertyConfigById: this.getPropertyConfiguration,
                 cardInteractionSupport: noCardInteractionsSupported,
+                windowUtils: null,
```

Comment:
I understand why these values are null, but it would be nice if we can come up with a way to avoid this.

Reply:
I think it's ok, the values should never be used in the report, so we should be fine